### ISSUE TITLE
Topbar not dropping down

### ISSUE THREAD

**Issue by visiblefrank** [milohuang/reverie#303]
Thanks for viewing.

First attempt with reverie and foundation and cannot get dropdown menu to drop down. Was working originally and can't work what I did to stop it working. Where can I start looking for problems with this function? 

Many thanks

**Comment by Grahamhoare** [comment]
Yes I get this problem every install, you have to put a line of code in that sais on hover of the top li the dropdown is visible, not hidden as in the standard setup, a real pain, the drop down should work automatically without tweaking a lot of people use it.
.top-bar-section ul li.has-dropdown:hover .dropdown {   visibility: visible;}
regards G

**Comment by visiblefrank** [comment]
Thanks Graham!
